tests/test_main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import pytest
+from app.main import get_human_age
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, human_age",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ],
+)
```

Comment:
As Sasha said, you should use here 
` pytest.param(
            [],
            [],
            id="Some description"
        ),`

Reply:
@MasterpieceElbow but at the time of this task, the use of the mentioned methods of solution has not yet been studied by us.